### ISSUE TITLE
crashing on startup

### ISSUE THREAD

**Issue by CaliMali** [akorb/SteamShutdown#42]
[SteamShutdown_Log.zip](https://github.com/akorb/SteamShutdown/files/6910704/SteamShutdown_Log.zip)

**Comment by Malkovitz** [comment]
it's crashing again, even on older version

**Comment by akorb** [comment]
I'm very sorry to hear this. Seems like Steam is changing a lot with the
files recently. But I didn't investigate the new problem yet.

Unfortunately, I don't have time at the moment. Probably not until mid of August,
maybe someone else could fix the problem in the meanwhile.

**Comment by akorb** [comment]
In the meanwhile, could you please upload your `libraryfolders.vdf` file? 
It's located in `..\Steam\SteamApps`

**Comment by CaliMali** [comment]
[libraryfolders.zip](https://github.com/akorb/SteamShutdown/files/6912420/libraryfolders.zip)

**Comment by Malkovitz** [comment]
[libraryfolder.zip](https://github.com/akorb/SteamShutdown/files/6912553/libraryfolder.zip)

**Comment by Malkovitz** [comment]
[libraryfolder.zip](https://github.com/akorb/SteamShutdown/files/6912554/libraryfolder.zip)
second one

**Comment by akorb** [comment]
@CaliMali
Could you please verify that this version works again?
[SteamShutdown.zip](https://github.com/akorb/SteamShutdown/files/6919499/SteamShutdown.zip)

@Malkovitz
Your files work for me. But I had to rename them from `libraryfolder.vdf` to `libraryfolders.vdf` to work (note the 's'). Also the old versions. Is the problem maybe still fixed for you somehow? If not, please open a new issue with the log file from your Desktop attached.

**Comment by akorb** [comment]
Just for completeness, I got a positive feeback via e-mail. I'd like to have a second positive feedback before I release a new version

**Comment by Malkovitz** [comment]
All good, works for me.

wt., 3 sie 2021 o 21:52 Andreas Korb ***@***.***> napisał(a):

**Comment by akorb** [comment]
Thanks, I just pushed a new release. I hope the problem is fixed now. If not, feel free to reopen this issue.

**Comment by CaliMali** [comment]
your version you tagged me with works, the latest push doesnt and crashes.

**Comment by akorb** [comment]
> 
> 
> your version you tagged me with works, the latest push doesnt and crashes.

I just checked it again as well. With the Release 2.3.7 the program crashes for me with your libraryfolders.vdf file. But with the new 2.3.8 release it works for me.
Is the log the same as before for you? Could you please validate, that you're really running 2.3.8? You can check it by right clicking `SteamShutdown.exe` and then going to `Details`.

It would really confuse me if this problem persists for you with the released version but does not with the testing version here. The only difference should be the version number.

**Comment by CaliMali** [comment]
> > your version you tagged me with works, the latest push doesnt and crashes.
> 
> I just checked it again as well. With the Release 2.3.7 the program crashes for me with your libraryfolders.vdf file. But with the new 2.3.8 release it works for me.
> Is the log the same as before for you? Could you please validate, that you're really running 2.3.8? You can check it by right clicking `SteamShutdown.exe` and then going to `Details`.
> 
> It would really confuse me if this problem persists for you with the released version but does not with the testing version here. The only difference should be the version number.

Apologies, 2.8.3 Does indeed work, i must've ran the wrong version of the release.. Thank you!

/closed
